packages/admin/cms-admin/src/builds/BuildEntry.tsx

```diff
@@ -105,14 +105,12 @@ const BuildStatusPopperContent: React.FunctionComponent<{ data: GQLBuildStatusQu
 
 export function BuildEntry(): React.ReactElement {
     const { data, error, refetch, startPolling, stopPolling } = useQuery<GQLBuildStatusQuery>(buildStatusQuery, {
-        skip: process.env.NODE_ENV === "development",
```

Comment:
We shouldn't change it here, since Kubernetes isn't available in development, and would cause an error every ten seconds.

Reply:
Alright, I removed the changes in this file altogether because 

- skip on the query alone doesn't work, skip on useFocusAwarePolling() is also needed
- I missed that this should apparently only be enabled in production